ReactViewControl/IViewModule.cs

```diff
@@ -1,12 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 
 namespace ReactViewControl {
 
     public interface IViewModule {
 
         string MainJsSource { get; }
 
+        IModuleDependenciesProvider DependenciesProvider { get; set; }
```

Comment:
should be readonly

Reply:
Hum... We can do that, but we will need to open an API to bind DependenciesProvider instance (when using hot-reload), since the implementations of IViewModule are generated

Reply:
I've used the already existing Bind API for that